python/arcticdb/version_store/_normalization.py

```diff
@@ -39,14 +37,8 @@
 
 PICKLE_PROTOCOL = 4
 
-try:  # 0.21+ Compatibility
-    from pandas._libs.tslib import Timestamp
-    from pandas._libs.tslibs.timezones import get_timezone
-except ImportError:
```

Comment:
I figure we can remove support for such old versions. Man doesn't use such old Pandas.